src/main/sensors/voltage.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define VBAT_SCALE_DEFAULT 110
+#define VBAT_RESDIVVAL_DEFAULT 10
+#define VBAT_RESDIVMULTIPLIER_DEFAULT 1
+#define VBAT_SCALE_MIN 0
+#define VBAT_SCALE_MAX 255
+
+#define VBATT_LPF_FREQ  1.0f
+
+#ifndef MAX_VOLTAGE_METERS
+#define MAX_VOLTAGE_METERS 1 // VBAT - some boards have external, 12V and 5V meters.
+#endif
+
+typedef struct voltageMeterConfig_s {
+    uint8_t vbatscale;                      // adjust this to match battery voltage to reported value
```

Comment:
only use is  

https://github.com/cleanflight/cleanflight/blob/62646021f2bc27f8404ba9ac620f5eb0f8f75a89/src/main/sensors/voltage.c#L70 : 
```
return ((((uint32_t)src * config->vbatscale * 33 + (0xFFF * 5)) / (0xFFF * config->vbatresdivval)) / config->vbatresdivmultiplier);
```
`33` is 10 * ADC reference voltage, `0xFFF * 5` was used to round correctly (does not make sense in current code)

IMO it is reasonable to specify scale as input divider ratio, probably in mV/V - 1000 for no divider, max value would be ~ 65V/V, 214V input range. Resolution will be better that 1/1000 (it makes little sense to use amplifier as input).

Whole equation will reduce to 
```
src / <input range> * 3.3V * vbatscale / 100  // result in 0.1V
```
With optimization and correct rounding (3,3V moved to denominator, accuracy reduction is negligible, floating point is eliminated in compiller). Also 1lsb is Vref+/4096, not 0xfff as in current code
```
((int)src * vbatscale + (int)(0x1000 * 100 / 3.3 / 2)) / (int)(0x1000 * 100 / 3.3);
```

Also I would prefer return value in mV (removing `* 100` in denominator)